app/e2e/client/tabbar.js

```diff
@@ -21,7 +21,13 @@ Meteor.startup(() => {
 					call('saveRoomSettings', room._id, 'encrypted', !room.encrypted);
 				},
 				order: 13,
-				condition: () => hasAllPermission('edit-room', Session.get('openedRoom')),
+				condition: () => {
+					const room = ChatRoom.findOne(Session.get('openedRoom'));
```

Comment:
Maybe a stupid "improvement", but I'd save `const rid = Session.get('openedRoom')` and use it where necessary, in order to avoid multiple Session accesses. Not sure if worth it.

Reply:
I don't see a problem in saving the value. Fixed